src/app/static/src/app/store/adr/actions.ts

```diff
@@ -154,9 +155,21 @@ export const actions: ActionTree<ADRState, RootState> & ADRActions = {
             const { resourceType, resourceFilename, resourceId } = uploadFilesPayload[i]
 
             const requestParams: Dict<string> = {resourceFileName: resourceFilename}
-            if (resourceId){
+            if (resourceId) {
                 requestParams["resourceId"] = resourceId
             }
+            if (uploadMetadata && resourceType === "inputs-unaids-naomi-report") {
```

Comment:
These resource types shouldn't be hard-coded. Use state.schemas.outputZip and .outputSummary